src/e2e/puppeteer/helpers/waitForTransitionEnd.ts

```diff
@@ -0,0 +1,14 @@
+import { page } from '../setup'
+
+/** Wait for a transition to finish. */
+async function waitForTransitionEnd(selector: string) {
```

Comment:
I would also prefer this to be an arrow function, but I see that a few of the other helpers are also defined this way, so this shouldn't be a blocking issue for the PR.

@raineorshine -- It would be nice if we refactor to consistently use arrow functions IMO (and enforce with ESLint).